src/fsharp/FSComp.txt

```diff
@@ -1428,7 +1428,8 @@ keywordDescriptionUntypedQuotation,"Delimits a untyped code quotation."
 3216,itemNotFoundDuringDynamicCodeGen,"%s '%s' not found in assembly '%s'. A possible cause may be a version incompatibility. You may need to explicitly reference the correct version of this assembly to allow all referenced components to use the correct version."
 3216,itemNotFoundInTypeDuringDynamicCodeGen,"%s '%s' not found in type '%s' from assembly '%s'. A possible cause may be a version incompatibility. You may need to explicitly reference the correct version of this assembly to allow all referenced components to use the correct version."
 descriptionWordIs,"is"
-notAFunction,"This value is not a function and cannot be applied."
+notAFunction,"The value is of type '%s', which is not a function type. A value is being passed to it as an argument as if it were a function. This is invalid."
```

Comment:
A shorter version would be:
"This value, of type '%s', is not a function, but an argument is being passed to it."